testing/benchmark/Makefile

```diff
@@ -54,7 +54,7 @@ init-aws-profile: ~/.aws/credentials
 	@touch $@
 
 terraform.tfvars:
-	@sed "s/USER/$(USER)/" $(TFVARS_SOURCE) > terraform.tfvars
+	@cp $(TFVARS_SOURCE) terraform.tfvars
```

Comment:
This will break `make terraform.tfvars` locally, since the current user won't be used for prefixes, I'd rather keep it as it was.

Reply:
Updated!